MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,147 @@
+
+* Enable cropping and visually select the crop region.
+* Manually entering the cropping area.
+
+N> You can also crop the image at a specific position with annotations. 
+
+## Handling the cropping tool
+
+The `Crop` method in the image editor control crop the image based on the `ImageCropType`. It allows users to enable or disable the cropping region displayed over the image, making it easier to visually choose the area for cropping.
+
+    * ImageCropType - Specifies the selection type for cropping the image.
+
+* The following code shows cropping the image to any desired size.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Free);
+
+{% endhighlight %}
+
+After the cropping area has been selected, the `SaveEdits` method is called, which in turn crops the selected region and displays the cropped image on the image editor.
+
+{% highlight C# %}
+
+imageEditor.SaveEdits();
+
+{% endhighlight %}
+
+After selecting the cropping area, if you decide to cancel the edits and revert back to the original image, you can use the `CancelEdits` method. It discard the changes made during the cropping process and restore the image to its initial state.
+
+{% highlight C# %}
+
+imageEditor.CancelEdits();
+
+{% endhighlight %}
+
+## Freehand Crop
+
+This type of crop allows you to freely select and crop a region of the image. You can manually outline the desired area using your cursor or touch input.The selection region can be customized by dragging and resizing an image.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Free);
+
+{% endhighlight %}
+
+## Original Crop
+
+With the original crop, the image is cropped to its original dimensions. This means that no aspect ratio and the cropped image retains its original width and height.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Original);
+
+{% endhighlight %}
+
+## Square Crop
+
+The square crop option allows you to crop the image into a perfect square shape.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Square);
+
+{% endhighlight %}
+
+## Ratio Crop
+
+The ratio crop enables you to crop the image with a specific aspect ratio. You can specify the desired ratio using the width and height values. The default ratio is 4:3 when using `ImageCropType.Ratio`, but you can change it by using the `Crop` method with the ratio parameter.  
+
+{% highlight C# %}
+
+imageEditor.Crop(16,9);
+
+{% endhighlight %}
+
+## Circle crop
+
+You can crop an image in a circular format, specifically for profile pictures, you can use the `Crop` method. This method ensures that the image is cropped into a perfect circle with a 1:1 ratio.
+
+* The following code shows cropping an image in circle format.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Circle);
+
+{% endhighlight %}
+
+## Ellipse crop
+
+To crop an image in an elliptical format, such as for profile pictures, you can use the `Crop` method. This allows for free-style cropping in an elliptical shape.
```

Comment:
To crop an image in an elliptical format you can use the `ImageCropType.Ellipse` crop type.  This allows for free-style cropping in an elliptical shape.